api/src/lib.rs

```diff
@@ -1570,33 +1606,27 @@ where
     }
 
     {
-        // Verification tokens legacy endpoint. Remove for API v3.
-        let mut verify_tokens_legacy =
-            tide::with_state(State::new(verify_db.clone(), token_db.clone()));
-
-        verify_tokens_legacy
-            .at(":id")
-            .with(ensure_vault_id)
-            .with(cors.clone())
-            .options(option_ok)
-            .get(get_verification_status)
-            .post(create_verification_token)
-            .put(verify_verification_token);
-
-        api.at("verify_tokens").nest(verify_tokens_legacy);
-
         // Verification tokens
         let mut verify =
             tide::with_state(State::new(verify_db.clone(), token_db.clone()));
 
         verify
```

Comment:
I think its fine to have `signed_pow_auth` and `add_auth_info` here because, at least in the extension, we only create the verification token *after* the vault (eg keypair) has been created.

Reply:
The page that the link coming from the email opens only has the secret right?

Reply:
you are right, i am wrong :)